### DISCUSSION TITLE
Installation errors

### DISCUSSION THREAD

**Discussion by sdpatil07** [GreenleafLab/ArchR#1792] [Questions / Documentation] [answered]
I have installed devtools and biomanagers then I tried to install ArchR but at the end of the installation it showed following messages. Thanks for your help in advance.

ERROR: dependencies ‘motifmatchr’, ‘chromVAR’, ‘ComplexHeatmap’ are not available for package ‘ArchR’ 
* removing ‘/usr/local/lib/R/site-library/ArchR’ 
Warning messages: 
1: In i.p(...) : 
  installation of package ‘DirichletMultinomial’ had non-zero exit status 
2: In i.p(...) : installation of package ‘Cairo’ had non-zero exit status 
3: In i.p(...) : 
  installation of package ‘TFBSTools’ had non-zero exit status 
4: In i.p(...) : 
  installation of package ‘ComplexHeatmap’ had non-zero exit status 
5: In i.p(...) : 
  installation of package ‘chromVAR’ had non-zero exit status 
6: In i.p(...) : 
  installation of package ‘motifmatchr’ had non-zero exit status 
7: In i.p(...) : 
  installation of package ‘/tmp/Rtmpb8JSAA/file6ea0561d6155/ArchR_1.0.2.tar.gz’ had non-zero exit status

**Comment by rcorces** [comment, marked as answer]
try to install each package individually and troubleshoot each installation individually. These are dependencies for ArchR but this isnt a problem with ArchR itself so I'm not able to help.

**Reply by sdpatil07** [reply]
thanks, i was able to solve them. there were errors in my Debian system.

**Reply by Wenxi-Zhang** [reply]
Hi! May I ask how you resolve it? I'm having the same problem
